### PR TITLE
feat: Evergreen content for EInk screens

### PR DESCRIPTION
**Pull request by cmaddox5**
**Asana task**: [Add evergreen content to bus and green line e-Ink screens](https://app.asana.com/0/1185117109217413/1200928169910241/f)

Evergreen content can now be added to Bus and GL EInk screens. Styles should match resolution requirements for these screens, but please check me on that. Also found that GL EInk did not have a corresponding test file. Went ahead and added that here.

Description

- [ ] Needs version bump?

### CODE REVIEW

File: assets/css/v2/bus_eink/evergreen/image.scss [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  background-color: #e4e6e1;
+  overflow: hidden;
+}
+
+.flex-one-medium-two-small__left,
+.flex-two-medium__left,
+.flex-two-medium__right {
+  .evergreen-content-image__container {
+    border-radius: 16px;
+    box-shadow: 0px 12px 24px 0px rgba(23, 31, 38, 0.25);
```
Comment (jzimbel-mbta) [inline assets/css/v2/bus_eink/evergreen/image.scss:11]: Let's double check with design on whether they want shadows or rounded corners on these in the e-ink apps.

**Review by jzimbel-mbta — COMMENTED** [review]
Looks good overall! You got the dimensions right—the "medium flex" slot on [bus](https://github.com/mbta/screens/blob/master/assets/css/v2/bus_eink/screen/normal.scss) and [GL](https://github.com/mbta/screens/blob/master/assets/css/v2/gl_eink/screen/normal.scss) e-ink screen templates is 1200x1164px. We just need to change the CSS selectors to apply those styles to the correct elements on the e-ink apps.

I'm not 100% sure on some other styling details like the drop shadow and rounded corners, but we can check with design on that. In general if you are ever unclear on how something should look, don't hesitate to consult Mindy and/or Adam.

One small bit that's missing is the addition of columns in our admin tool to allow configuring evergreen content on bus and GL e-ink. I'll show you where to make that change during our meeting.

**Comment by handorff** [comment]
re: adding the columns to the admin tool: This is my bad, I put that in the Asana task description as one of the requirements but the column is actually already there. (It's part of `v2Columns` in `admin_tables.tsx`, which are included in both the Bus e-Ink and Green Line e-Ink v2 configs.)

File: assets/css/v2/bus_eink/evergreen/image.scss (hunk 1 above)
Comment (cmaddox5) [reply assets/css/v2/bus_eink/evergreen/image.scss:8]: Checked with Mindy on this. Border radius on evergreen e-ink is 32px and no drop shadow.

File: assets/css/v2/bus_eink/evergreen/image.scss [hunk 2]
```diff
@@ -0,0 +1,65 @@
+.evergreen-content-image__container {
+  overflow: hidden;
+}
+
+.screen-normal__medium-flex {
+  .evergreen-content-image__container {
+    border-radius: 16px;
+    box-shadow: 0px 12px 24px 0px rgba(23, 31, 38, 0.25);
```
Comment (jzimbel-mbta) [inline assets/css/v2/bus_eink/evergreen/image.scss:8]: Looks like you just need to make this change, based on Mindy's feedback:
```suggestion
    border-radius: 32px;
```

File: assets/css/v2/gl_eink/evergreen/image.scss [hunk 3]
```diff
@@ -0,0 +1,77 @@
+.evergreen-content-image__container {
+  overflow: hidden;
+}
+
+.screen-normal__medium-flex {
+  .evergreen-content-image__container {
+    border-radius: 16px;
+    box-shadow: 0px 12px 24px 0px rgba(23, 31, 38, 0.25);
```
Comment (jzimbel-mbta) [inline assets/css/v2/gl_eink/evergreen/image.scss:8]: Here as well:
```suggestion
    border-radius: 32px;
```

**Review by jzimbel-mbta — APPROVED** [review]

**Pull request merged**